portfolio/src/main/webapp/scripts/github/client.js

```diff
@@ -0,0 +1,16 @@
+const baseURL = "https://api.github.com";
```

Comment:
nit: generally use single instead of double quotes (https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes)

Same applies for other files

Reply:
Great, usually I use single quotes, but it turns out that my default Eslint configuration in VSCode changes them by double quotes.

I changed my configuration, so it applies formatting on saving, and with single quotes.

Just as a proposal, do you think that would be nice if I add a .eslintrc?

Reply:
Yeah, feel free to add an .eslintrc if you're up for it, sounds like a great idea!

Reply:
Cool, I'll add it in the next Pull Request.

Thanks!